pennylane/shadows/utils.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions common to classical shadows"""
```

Comment:
Thinking about moving these into the `classical_shadow.py` file to have everything in the same file and since these functions are rather specific to our implementation. Thoughts?

Reply:
I'm personally in favour of leaving these two utility functions here. Curious if @josh146 has a preference.

Reply:
I agree with moving it to `classical_shadow.py` here, mostly because I am not a fan of `utils.py` modules --- they tend to become 'misc' modules that are often forgotten about, and it is hard to reason what is in them solely from the name. We have been attempting to get rid of the `pennylane/utils.py` module for a while now 🙂

Reply:
Okay I can do the move 👍